runtime/doc/options.txt

```diff
@@ -4761,14 +4761,13 @@ A jump table for the options with a short description can be found at |Q_op|.
 	height with ":set scroll=0".
 
 						*'scrollback'* *'scbk'*
-'scrollback' 'scbk'	number	(default: 10000
-				 in normal buffers: -1)
+'scrollback' 'scbk'	number	(default: -1)
 			local to buffer
 	Maximum number of lines kept beyond the visible screen. Lines at the
 	top are deleted if new lines exceed this limit.
-	Only in |terminal| buffers.
-	-1 means "unlimited" for normal buffers, 100000 otherwise.
-	Minimum is 1.
+	Minimum is -1, maximum is 100000.
+	For terminal buffers: -1 is equivalent to 10000.
```

Comment:
This is a regression, terminal buffers hitherto supported 100k lines.

Reply:
That's still the case, 100k max, but 10k is the default. Actually, changing it to 100k would make a lot of tests fail.

Reply:
But this help says `-1 is equivalent to 10000`. If the max is 100k, why would `-1` go to only 10k?

Reply:
No idea, but that's how it worked before this patch as well. :> I figured 100k by default would take too much memory or something like that.